vite.config.mjs

```diff
@@ -106,7 +106,6 @@ export default defineConfig(async ({ mode }) => {
   const workersSjs = isProduction
     ? path.resolve(workersScalaClassesDir, 'workers-opt')
     : path.resolve(workersScalaClassesDir, 'workers-fastopt');
-  const rollupPlugins = isProduction ? [] : [visualizer()];
```

Comment:
unrelated, but visualizer was removed recently, and `rollupPlugins` isn't used anywhere.

Reply:
👍